templates/prometheus.yaml

```diff
@@ -84,6 +84,9 @@ spec:
                 targetPort: 5601
                 interval: 30s
         prometheusSpec:
+          thanos:
```

Comment:
are we gonna run thanos by default ?

Reply:
This is fine to me. Thanos is just a sidecar exposing a GRPC service. It won't do anything if no one is hitting the API I believe.

Reply:
I just wanted to get a confirmation. Thanks